app/assets/stylesheets/subjects.css

```diff
@@ -0,0 +1,4 @@
+/*
+  Place all the styles related to the matching controller here.
+  They will automatically be included in application.css.
+*/
```

Comment:
Probably don't need this file either. At least not until we actually start styling.